ci/code_checks.sh

```diff
@@ -259,17 +259,18 @@ fi
 if [[ -z "$CHECK" || "$CHECK" == "doctests" ]]; then
 
     MSG='Doctests frame.py' ; echo $MSG
-    pytest -q --doctest-modules pandas/core/frame.py
+    pytest -q --doctest-modules pandas/core/frame.py \
+        -k"-differences"
```

Comment:
Is this excluding the differences test? Wasn't expecting a change like this

Reply:
@WillAyd no I didn't intend to exclude them from the tests. I wanted to include them in the tests but got a bit confused myself. Could you take a look again?